### ISSUE TITLE
Release v1.0.0 requirements 🎯

### ISSUE THREAD

**Issue by LeeDongGeon1996** [DaveSkender/Stock.Indicators.Python#189]
Only few more steps are left for official v1.0.0 release.

Here's the checklist for releasing v1.0.0:
(Please feel free to add more items)

**Features**
 - [ ] Add remaining indicators.
 - [ ] Update existing indicators.
 - [ ] Final API check.

**Tests**
 - [ ] Update benchmarks.(#)
 - [ ] Add benchmark result to docs.
 - [ ] Add return `type` check.
 - [ ] Check if there's something more to test.

**Documentation**
 - [ ] Write corresponding documentation (# )
 - [ ] Final review for docs and docstrings.
 
**Release**
 - [ ] Check DLL version.
 - [ ] Final sanity check and code style.
 - [ ] Release to PyPI.
 - [ ] Apply release description to tag.

**Comment by DaveSkender** [comment]
I think a key element here, before we flip it to v1.0.0 is to ensure that it’s stable.  There’s a couple ways to do this:

- dogfood it.  Use the library in your own use case for a while, at least a couple months.
- just wait, for more user feedback.  This could take 3-6 months, depending on how much usage

**Comment by LeeDongGeon1996** [comment]
Yeah, the silver lining is that we are gradually getting more users according to Google Analytics.
